crates/ruff/src/registry.rs

```diff
@@ -754,6 +757,9 @@ pub enum Linter {
     /// [flake8-use-pathlib](https://pypi.org/project/flake8-use-pathlib/)
     #[prefix = "PTH"]
     Flake8UsePathlib,
+    /// [flake8-variables-names](https://pypi.org/project/flake8-variables-names/)
+    #[prefix = "VN"]
```

Comment:
Nit: I think these should be `VNE` to match the upstream plugin -- or was this an intentional deviation?

Reply:
I originally had that but changed it due to the few above not having E in their prefix, but makes sense to keep it to the original 🙂